platform-api-client-gwt/che-core-client-gwt-user/src/main/java/org/eclipse/che/api/user/gwt/client/UserServiceClientImpl.java

```diff
@@ -97,8 +97,8 @@ public void getUserById(@NotNull String id, AsyncRequestCallback<UserDescriptor>
 
     /** {@inheritDoc} */
     @Override
-    public void getUserByEmail(@NotNull String email, AsyncRequestCallback<UserDescriptor> callback) {
-        String requestUrl = FIND + "?email=" + email;
+    public void getUserByAlias(@NotNull String alias, AsyncRequestCallback<UserDescriptor> callback) {
+        String requestUrl = FIND + "?alias=" + alias;
```

Comment:
if alias is `null` it will be added to the params, which means that user will get a message _"User with alias `null` doesn't exist"_ when he should get kind of  _"'alias' query parameter is required"_, you should add alias only if it is not null.